conda_build/config.py

```diff
@@ -408,5 +412,19 @@ def show(config):
 # legacy exports for conda
 croot = Config().croot
 
+
+class Context(Configuration):
```

Comment:
Should the whole Config class become a Configuration subclass?  I'm thinking towards allowing people to set more permanent defaults for conda-build's other settings.